app/javascript/courses/review/components/CoursesReview__ChecklistEditor.re

```diff
@@ -297,7 +299,9 @@ let make =
                             <School__InputGroupError
                               message="Title should be greater than 2 characters"
                               active={
-                                resultItem |> ReviewChecklistResult.title == ""
+                                reviewChecklistItem
+                                |> ReviewChecklistItem.title
+                                |> String.length <= 2
```

Comment:
These checks should have been against `resultItem`, not `reviewChecklistItem`.

Reply:
Oh derp! 

Sorry about that, have made the change